Marlin/ultralcd_impl_HD44780.h

```diff
@@ -193,12 +193,18 @@ extern volatile uint8_t buttons;  //an extended version of the last checked butt
   static void lcd_implementation_update_indicators();
 #endif
 
-static void lcd_set_custom_characters(
-  #if ENABLED(LCD_PROGRESS_BAR)
-    const bool info_screen_charset = true
-  #endif
-) {
-  static byte bedTemp[8] = {
+
+static void createChar_P(char c, PROGMEM byte *ptr) {
+  byte temp[8];
+  int8_t i;
+
+  for(i=0; i<8; i++)  {
```

Comment:
- One space between `for` and the opening parenthesis.
- Pad operators with spaces, including `=` and `<` within a `for`.

If we don't uphold and exemplify the [Coding Standards](http://marlinfw.org/docs/development/coding_standards.html) then it sets a bad example for the newbies.